certs/swagger.yml

```diff
@@ -19,21 +19,21 @@ paths:
       parameters:
       - $ref: "#/parameters/Authorization"
       - in: body
-         name: cert
-         description: 
-         schema:
-           type: object
-           required:
-             - thing_id
-             - days_valid
-             - rsa_bits
-           properties:
-              thing_id:
-                type: string
-              days_valid:
-                type: string
-              rsa_bits:
-                type: integer
+        name: cert
+        description: TODO
```

Comment:
TODO :)

Reply:
It's OK to leave `TODO`, especially if the validator demands something to be written as a value to a key.

But if we can add description, that would be better.

Reply:
@mteodor has to fix this, since I don't know what the service does exactly.

Reply:
@darkodraskovic  can you add 
 ```description: |
        Creates a certificate that is required for mTLS. To create a certificate for a thing
        provide a thing id, data identifying particular thing will be embedded into the Certificate.
        x509 and ECC certificates are supported.```